wikimedia/wikipedia-ios
# Native Editor - Add basic undo / redo support

**Phabricator:**
https://phabricator.wikimedia.org/T348088

### Notes
This PR connects the undo / redo buttons to the editor's undo manager. It also disables the progress button until the text has changed.

### Test Steps
1. On Staging, open an article editor. Undo, Redo, and Next buttons should be disabled.
2. Make a change in the text view. Confirm undo button enables.
3. Tap undo button. Confirm change is undone.
4. Tap redo button. Confirm change is redone.
5. Confirm Next button only enables when there are changes to publish.
